projects/ngrx.io/content/guide/store-devtools/recipes/exclude.md

```diff
@@ -2,52 +2,68 @@
 
 To prevent Store Devtools from being included in your bundle, you can exclude it from the build process.
 
+## Step 1: Put Store Devtools In `environment.ts`
 
-## Step 1: Create build specific files
+To exclude it, put `@ngrx/store-devtools` into an `environment.ts` file, which is replaced with `environment.prod.ts`. Given the below example:
```

Comment:
Can we add a note here on how to generate the environments (since they are not included in new projects)?


```suggestion
To exclude the DevTools, put `@ngrx/store-devtools` into an `environment.ts` file, which is replaced with `environment.prod.ts`. 

<div class="alert is-helpful">

If the environment files don't exist in your project you can use the `ng generate environments` command to create them.

</div>

Given the below example:
```